toc/ROLES.md

```diff
@@ -65,7 +79,10 @@ establish sub-working groups. Working groups delegate change approval to Approve
   <tr>
     <td>Approver</td>
     <td>
-        <p>Approve contributions from other members</p>
+        <p><b>For 1+ repos in an area:</b></p>
+        <p>Triage PRs and issues</p>
```

Comment:
For consistency, these additions should also be stated in the Approver section later on in the document.

Reply:
Added a suggestion to add this line to the Approver section below.